grails-app/conf/Config.groovy

```diff
@@ -1504,6 +1504,16 @@ openboxes {
                                     [label: "react.putAway.createPutAway.label", defaultLabel: "Create Putaway", href: "/${appName}/putAway/create"],
                                     [label: "react.putAway.list.label", defaultLabel: "List Putaways", href: "/${appName}/order/list?orderType=PUTAWAY_ORDER&status=PENDING"]
                             ]
+                    ],
+                    [
+                            label: "receiving.label",
+                            defaultLabel: "Receiving",
+                            enabled: false, // DEPRECATED!
```

Comment:
Could you move the `DEPRECATED` comment above the label? Currently, it suggests that the `enabled` field is deprecated.